### PR TITLE
[CB-175] Results form features number of results

### PR DESCRIPTION
**Pull request by LWanjiru**
# JIRA TICKET NAME:
CB-175 - [Results form features number of results
](https://issues.openmrs.org/browse/CB-175)

## SUMMARY:
**Currently,** when a user clicks on the results link, they view the results with a heading detailing only the filters applied to produce those results.

**On fix**, a user should be able to see how may results the search yielded on the result form.

### CODE REVIEW

File: app/js/components/common/table.jsx [hunk 1]
```diff
@@ -77,6 +77,7 @@ class cohortTable  extends Component {
         {this.state.toDisplay.length > 0 ?
           <div className="table">
             <h1 className="text-center">{this.state.description}</h1>
+            <h5 className="text-center">This search has {this.state.searchHistory.length} result(s)</h5>
```
Comment (andela-dbamidele) [inline app/js/components/common/table.jsx]: You can also deconstruct `this.state` to reduce the line length.

**Review by andela-dbamidele — COMMENTED** [review]
Great job. 👍

File: app/js/components/common/table.jsx (hunk 1 above)
Comment (LWanjiru) [reply app/js/components/common/table.jsx]: Thank you for that observation. Done!

**Comment by joel-ace** [comment]
Looks Good

**Pull request merged**